railties/test/generators/plugin_generator_test.rb

```diff
@@ -258,6 +258,7 @@ def test_creating_engine_in_full_mode
   def test_creating_engine_with_hyphenated_name_in_full_mode
     run_generator [File.join(destination_root, "hyphenated-name"), "--full"]
     assert_no_file "hyphenated-name/app/assets/javascripts/hyphenated/name"
+    assert_no_file "hyphenated-name/test/dummy/app/javascript"
```

Comment:
I think this assertion should add to `test_unnecessary_files_are_not_generated_in_dummy_application` and unneed to add  to other tests.

Reply:
Okay. I'll fix it.

Reply:
@y-yagi Done. Please have a look